satpy/readers/hrit_base.py

```diff
@@ -117,28 +120,31 @@ class HRITFileHandler(BaseFileHandler):
 
     def __init__(self, filename, filename_info, filetype_info, hdr_info):
         """Initialize the reader."""
-        super(HRITFileHandler, self).__init__(filename, filename_info,
-                                              filetype_info)
+        super().__init__(filename, filename_info, filetype_info)
 
         self.mda = {}
         self.hdr_info = hdr_info
         self._get_hd(self.hdr_info)
         self._start_time = filename_info["start_time"]
         self._end_time = self._start_time + dt.timedelta(minutes=15)
 
-    def _get_hd(self, hdr_info):
+    def _get_hd(self, hdr_info, verbose=False):
         """Open the file, read and get the basic file header info and set the mda dictionary."""
         hdr_map, variable_length_headers, text_headers = hdr_info
-
         with utils.generic_open(self.filename, mode="rb") as fp:
             total_header_length = 16
             while fp.tell() < total_header_length:
                 hdr_id = get_header_id(fp)
+                if verbose:
```

Comment:
Is it possible to use a logger for this?

Reply:
well, the idea is that we use this when we need to generate stub files, so I don't think it should be in the logging...

Reply:
ok!